lazyflow/operators/ioOperators/opInputDataReader.py

```diff
@@ -301,7 +301,7 @@ def _attemptOpenAsOmeZarrUri(self, filePath):
             return ([], None)
         if not isUrl(filePath):
             filePath = Path(filePath).as_uri()
-        if not (filePath.startswith("http") or filePath.startswith("file")):
+        if not any(filePath.startswith(scheme) for scheme in ["http", "file", "s3"]):
```

Comment:
it would be a bit more in tune with the rest of the file to declare a class variable in `OpInputDataReader` that lists these (same for precomputed scheme). Also is it will the scheme not always include `://` as well?

Reply:
The `:` would be guaranteed, but file URIs don't necessarily continue with a / after that.